cmd/ocm-backplane/login/login.go

```diff
@@ -183,6 +185,13 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	}
 
 	logger.Debugf("Using backplane URL: %s\n", bpURL)
+	backplaneResolution, err := getBackplaneCNAME(bpURL)
+	if err != nil {
+		logger.Warn(err.Error())
+	} else {
+		logger.Debugf("Backplane URL resolves to %s \n", backplaneResolution)
+		logger.Debugf("Please refer to your design documentation to figure out the associated backplane server")
```

Comment:
Why exactly do we need this to be logged? Is there is known reasoning behind it?

Reply:
The main purpose of the ticket was to provide a way for SRE to figure out which back plane clusters the user is connecting to. Since providing the back plane server name is not possible due to the architecture, we just lookup for the ELB and then log it here. 
Once the ELB is known, the SRE can refer to the GitLab/Source Doc to figure out the exact backplane server.

Reply:
Added a follow-up suggestion